packages/react-components/package.json

```diff
@@ -13,28 +13,49 @@
     "clean": "tsc --build --clean",
     "test": "jest --watch",
     "test:coverage": "jest --ci --coverage",
+    "test:karma": "./node_modules/karma/bin/karma start karma.conf.js",
```

Comment:
nit:
Is it possible to use the karma cli?

Reply:
we can use it if we install karma globally which is no ideal.